addons/website/static/src/js/website.js

```diff
@@ -310,13 +310,18 @@
         $(document).on('click', '.js_publish_management .js_publish_btn', function () {
             var $data = $(this).parents(".js_publish_management:first");
             var self=this;
-            openerp.jsonRpc($data.data('controller') || '/website/publish', 'call', {'id': +$data.data('id'), 'object': $data.data('object')})
-                .then(function (result) {
-                    $data.toggleClass("css_unpublished css_published");
-                    $data.parents("[data-publish]").attr("data-publish", +result ? 'on' : 'off');
-                }).fail(function (err, data) {
-                    website.error(data, '/web#return_label=Website&model='+$data.data('object')+'&id='+$data.data('id'));
-                });
+            $('.btn-group').ready( function () {
```

Comment:
i tried it with .on('click') button action. but it required two click action on toggle button. so, i changed it to .ready()